umap/tests/test_map_views.py

```diff
@@ -818,10 +818,10 @@ def test_oembed_map(client, map, datalayer):
 def test_oembed_link(client, map, datalayer):
     response = client.get(map.get_absolute_url())
     assert response.status_code == 200
+
     assert (
-        '<link rel="alternate" type="application/json+oembed"'
-        in response.content.decode()
-    )
+        '<link rel="alternate"\n        type="application/json+oembed"'
```

Comment:
Not a big fan of having spaces in the tests here. Any suggestion welcome :-)

Reply:
Maybe applying [dedent](https://docs.python.org/3/library/textwrap.html#textwrap.dedent) to the deocded response?